triplea_maps.yaml

```diff
@@ -2063,3 +2063,31 @@
     <br>
     <br> This is the Charles de Gaulle 1939 Mod for the "Global 40" game.
     <br>
+- mapName: Twelve-Clans
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/twelve-clans/archive/master.zip
+  version: 1.0
```

Comment:
I think the version here needs to be "1" instead of "1.0". The numbers in this file don't necessarily correspond to the map's actual version and are just used by the TripleA engine to determine when players should be prompted that there is an update for the map. And yes, I realize its a bit confusing :)

Reply:
OK